<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchFriend</class>
 <widget class="QDialog" name="searchFriend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Friend</string>
  </property>
  <widget class="QPushButton" name="sendFriend">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Send friend request</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelRequest">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameFriend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Your friend's name</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
